common/flatpak-installed-ref.c

```diff
@@ -52,6 +52,7 @@ struct _FlatpakInstalledRefPrivate
   char    *appdata_name;
   char    *appdata_summary;
   char    *appdata_version;
+  char    *appdata_license;
```

Comment:
This needs a `g_free` in the `_finalize()` (and similarly for some of the other properties)